### PR TITLE
updated models

### PR DESCRIPTION
**Pull request by jasongoeble**
- updated the user table to only required fields for registration

- updated the profile table to contain all additional fields (which we all agreed were fields to have on the profile page, per project proposal document)

- removed the extra user table

- tested server startup and correct table generation from models --> good

### CODE REVIEW

File: server/models/profile.js [hunk 1]
```diff
@@ -1,22 +1,97 @@
...
+      {
+          type: DataTypes.STRING,
+          allowNull: true
+      },
+      avatar:
+      {
+          type: DataTypes.BLOB,
+          allowNull: true
+      },
+      //need query place averaged value of entries from reputation into this field
```
Comment (dylanburkey) [inline server/models/profile.js]: How will reputation work? I am just curious for academic reasons.

Comment (jasongoeble) [reply server/models/profile.js]: this field is purely for future implementation, based on the agreed upon desire to have a review of user process.  there is currently no work flow for it and it is not a required field in the database.  

conceptually, my thought was that when someone reviews you (numeric scale, capturing both reviewer and reviewee) then we could run a stored procedure to take the weighted average of all reviews and populate that field.

**Comment by jasongoeble** [comment]
made additional model tweaks

**Comment by jasongoeble** [comment]
didn't mean to delete

**Pull request merged**